ui/smite_priest/presets.ts

```diff
@@ -43,297 +43,32 @@ export const DefaultConsumes = Consumes.create({
-					Gems.RUNED_ORNATE_RUBY,
-					Gems.RUNED_LIVING_RUBY,
-					Gems.RUNED_LIVING_RUBY,
-				],
-			}),
-			ItemSpec.create({
-				id: 30666, // Ritssyn's Lost Pendant
-			}),
-			ItemSpec.create({
-				id: 21869, // Frozen Shadoweave Shoulders
-				enchant: Enchants.GREATER_INSCRIPTION_OF_DISCIPLINE,
-				gems: [
-					Gems.RUNED_LIVING_RUBY,
-					Gems.RUNED_LIVING_RUBY,
-				],
-			}),
-			ItemSpec.create({
-				id: 28570, // Shadow-Cloak of Dalaran
-				enchant: Enchants.SUBTLETY,
-			}),
-			ItemSpec.create({
-				id: 21871, // Frozen Shadoweave Robe
-				enchant: Enchants.CHEST_EXCEPTIONAL_STATS,
-				gems: [
-					Gems.RUNED_LIVING_RUBY,
-					Gems.RUNED_LIVING_RUBY,
-				],
-			}),
-			ItemSpec.create({
-				id: 24250, // Bracers of Havok
-				enchant: Enchants.WRIST_SPELLPOWER,
-				gems: [
-					Gems.RUNED_LIVING_RUBY,
-				],
-			}),
-			ItemSpec.create({
-				id: 28507, // Handwraps of Flowing Thought
-				enchant: Enchants.GLOVES_SPELLPOWER,
-				gems: [
-					Gems.RUNED_LIVING_RUBY,
-					Gems.RUNED_LIVING_RUBY,
-				],
-			}),
-			ItemSpec.create({
-				id: 28799, // Belt of Divine Inspiration
-				gems: [
-					Gems.RUNED_LIVING_RUBY,
-					Gems.RUNED_LIVING_RUBY,
-				],
-			}),
-			ItemSpec.create({
-				id: 24262, // Spellstrike Pants
-				enchant: Enchants.RUNIC_SPELLTHREAD,
-				gems: [
-					Gems.RUNED_LIVING_RUBY,
-					Gems.RUNED_LIVING_RUBY,
-					Gems.RUNED_LIVING_RUBY,
-				],
-			}),
-			ItemSpec.create({
-				id: 21870, // Frozen Shadoweave Boots
-				enchant: Enchants.BOARS_SPEED,
-				gems: [
-					Gems.RUNED_LIVING_RUBY,
-					Gems.RUNED_LIVING_RUBY,
-				],
-			}),
-			ItemSpec.create({
-				id: 29352, // Cobalt Band of Tyrigosa
-				enchant: Enchants.RING_SPELLPOWER,
-			}),
-			ItemSpec.create({
-				id: 28793, // Band of Crimson Fury
-				enchant: Enchants.RING_SPELLPOWER,
-			}),
-			ItemSpec.create({
-				id: 28789, // Eye of Magtheridon
-			}),
-			ItemSpec.create({
-				id: 29370, // Icon of the Silver Crescent
-			}),
-			ItemSpec.create({
-				id: 29350, // The Black Stalk
-			}),
-			ItemSpec.create({
-				id: 28770, // Nathrezim Mindblade
-				enchant: Enchants.SOULFROST,
-			}),
-			ItemSpec.create({
-				id: 29272, // Orb of the Soul-Eater
-			}),
-		],
-	}),
+	gear: EquipmentSpec.fromJsonString('{ "items":[ { "id":24266, "enchant":29191, "gems":[ 28118, 24030, 24030 ] }, { "id":28530 }, { "id":29060, "enchant":28886, "gems":[ 24030, 24030 ] }, { "id":28766, "enchant":33150 }, { "id":29056, "enchant":24003, "gems":[ 24030, 24030, 24030 ] }, { "id":24250, "enchant":22534, "gems":[ 24030 ] }, { "id":30725, "enchant":28272, "gems":[ 24030, 24030 ] }, { "id":24256, "gems":[ 24030, 24030 ] }, { "id":30734, "enchant":24274, "gems":[ 24030, 24030, 24030 ] }, { "id":28517, "enchant":35297, "gems":[ 24030, 24030 ] }, { "id":28793, "enchant":22536 }, { "id":29172, "enchant":22536 }, { "id":27683 }, { "id":29370 }, { "id":30723, "enchant":22555, "gems":[ 30564, 31867 ] }, { "id":28734 }, { "id":28673 } ]}'),
```

Comment:
Would it be possible to use an online pretty-printer site (e.g. https://jsonformatter.org/json-pretty-print) and make this a multi-line string? That way its at least slightly readable.

Same goes for the other presets.

Reply:
100% me being lazy and collapsing the json instead of thinking to lookup how to do multi-line string in go (which is more simple) heh